mesop/labs/chat.py

```diff
@@ -169,7 +169,7 @@ def chat(
   """
   state = me.state(State)
 
-  def on_submit(e: me.ClickEvent | me.EnterEvent):
+  def on_submit(e: me.ClickEvent | me.InputEnterEvent):
```

Comment:
Are you planning to remove the on input handler as a follow up? I think if on input is removed, then InputEnterEvent won't work as is since we'd need to set the input here.

Reply:
Yeah, good point, I'll take care of this in a follow-up PR.